org.eclipse.jdt.ui/ui/org/eclipse/jdt/internal/ui/text/correction/ReorgCorrectionsSubProcessor.java

```diff
@@ -595,7 +568,34 @@ public ICommandAccess createProjectSetupFixProposal(IInvocationContextCore conte
 
 	@Override
 	public boolean addImportNotFoundProposals(IInvocationContextCore context, IProblemLocationCore problem, Collection<ICommandAccess> proposals) throws CoreException {
-		return ReorgCorrectionsSubProcessor.importNotFoundProposals(context, problem, proposals);
+		ICompilationUnit cu= context.getCompilationUnit();
```

Comment:
Why doesn't this use the new base method?

Reply:
Fixed.